scanpy/tools/_dpt.py

```diff
@@ -743,11 +747,7 @@ def _detect_branching(
         tips: np.ndarray,
         seg_reference=None,
     ) -> Tuple[
-        List[np.ndarray],
-        List[np.ndarray],
-        List[List[int]],
-        List[List[int]],
-        int,
+        List[np.ndarray], List[np.ndarray], List[List[int]], List[List[int]], int
```

Comment:
```suggestion
        List[np.ndarray],
        List[np.ndarray],
        List[List[int]],
        List[List[int]],
        int,
```

I think this is fine. Seems intentional, due to the trailing comma.